packages/core/integration-tests/test/integration/commonjs-require-template-literals/index.js

```diff
@@ -0,0 +1,7 @@
+var local = require(`./local`);
+// eslint-disable-next-line no-unused-vars
+var url = require('url');
```

Comment:
Was there a reason you imported `url`? It's unused anyway

Reply:
Not really; it was just copied over from `commonjs` example. I thought it was some sort of _regression test_ of successfully loading the internal packages (with no errors).
I will address this as well as it doesn't make sense to be present here.